package.json

```diff
@@ -28,6 +28,7 @@
     "node-sass": "^4.5.3"
   },
   "dependencies": {
+    "intro.js": "file:node_modules/intro.js",
```

Comment:
I thought you did `npm install --save intro.js`. Which by no chance would include a dependency like this. Fix it.